src/markdown/templatemark.cto

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace org.accordproject.templatemark
+
+import org.accordproject.commonmark.Node from https://models.accordproject.org/markdown/commonmark@0.2.0.cto
+import org.accordproject.commonmark.Child from https://models.accordproject.org/markdown/commonmark@0.2.0.cto
+
+/**
+ * A model for Accord Project template extensions to commonmark
+ */
+abstract concept ElementDefinition extends Child {
+  o String name // Add Concerto regex
```

Comment:
// Add Concerto regex

Reply:
🙄 Are those like JS regex? Do we have some documentation?

Reply:
Example:
o String V5cID regex=/^[A-z][A-z][0-9]{7}/

I added it to the unstable docs: https://docs.accordproject.org/docs/next/model-properties.html

Reply:
That's probably why developers never write comments 😀